modules/docs/arrow-docs/docs/docs/arrow/typeclasses/semiring/README.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I tried to update everything as you suggested, but I'm still doing something wrong. Ank is able to compile everything without errors but when running the Ank playground examples, it fails because of unresolved references to `Int.semiring()`.  I guess it is basically the same problem as [here](https://arrow-kt.io/docs/arrow/typeclasses/show/) in the playground example. If I change `semiring` with `semigroup` everything runs without problems. I also checked if the classpath of the docs module contains an up to date arrow-core-extensions.jar, because the `Int.semiring()` extension function is declared inside.
Any ideas?

Reply:
@raulraja I just figured out that Ank has nothing to do with the actual playground snippet (:man_facepalming:), so I guess the problem is that the arrow playground server doesn't know about my changes yet. Sorry, my bad..